modules/@apostrophecms/import-export-piece-type/index.js

```diff
@@ -36,7 +36,10 @@ module.exports = {
         export: {
           label: 'aposImportExport:export',
           messages: {
-            progress: 'aposImportExport:exporting'
+            progress: 'aposImportExport:exporting',
+            completed: 'aposImportExport:exported',
+            icon: 'database-export-icon',
```

Comment:
already by default in the core, which is weird and too close to the `piece-type-exporter` module.

still set it here because it's this module responsibility to set that database export icon